rllib/core/learner/learner_group.py

```diff
@@ -557,6 +551,197 @@ def _load_state(w):
 
             self._worker_manager.foreach_actor(_load_state, remote_actor_ids=workers)
 
+    def load_module_state(
+        self,
+        *,
+        marl_module_ckpt_dir: Optional[str] = None,
+        modules_to_load: Optional[Set[str]] = None,
+        rl_module_ckpt_dirs: Optional[Mapping[ModuleID, str]] = None,
+    ) -> None:
+
+        """Load the checkpoints of the modules being trained by this LearnerGroup.
+
+        `load_module_state` can be used 3 ways:
+            1. Load a checkpoint for the MultiAgentRLModule being trained by this
+                LearnerGroup. Limit the modules that are loaded from the checkpoint
+                by specifying the `modules_to_load` argument.
+            2. Load the checkpoint(s) for single agent RLModules that
+                are in the MultiAgentRLModule being trained by this LearnerGroup.
+            3. Load a checkpoint for the MultiAgentRLModule being trained by this
+                LearnerGroup and load the checkpoint(s) for single agent RLModules
+                that are in the MultiAgentRLModule. The checkpoints for the single
+                agent RLModules take precedence over the module states in the
+                MultiAgentRLModule checkpoint.
+
+        NOTE: At lease one of multi_agent_module_state or single_agent_module_states
```

Comment:
I don't get this NOTE: there is no `multi_agent_module_state` or `single_agent_module_states` in the args of this method.

Reply:
These may be left overs of your dev history

Reply:
fixed.